tests/test_non_uniform_CMFD_MPI/test_non_uniform_CMFD_MPI.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import glob
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+from testing_harness import TestHarness
+import openmoc
+from openmoc.log import py_printf
+import subprocess
+
+
+class TestNonUniformCMFDMPITestHarness(TestHarness):
+    """A test of non-uniform CMFD OpenMOC running in parallel."""
+
+    def __init__(self):
+        super(TestNonUniformCMFDMPITestHarness, self).__init__()
+
+    def _create_geometry(self):
+        pass
+
+    def _create_trackgenerator(self):
+        pass
+
+    def _generate_tracks(self):
+        pass
+        
+    def _creat_solver(self):
+        pass
+
+    def _run_openmoc(self):
+      pwd = os.getcwd()
+      cmd = 'cp ../../profile/models/run_time_standard/run_time_standard.cpp ' \
+            + pwd
+      cmd += ' && ' + 'cp ../../profile/models/run_time_standard/' + \
+            'non-uniform-lattice.geo ' + pwd
+      rc = subprocess.call(cmd, shell=True)
+      
+       # Check for error code
+      if rc != 0:
+        print('\nERROR: Failed to copy run_time_standard.cpp or \
+              non-uniform-lattice.geo.\n')
+        os._exit(0)
+      
+      
+      os.chdir("../../profile/")
+      case = pwd + '/run_time_standard.cpp'
+      cmd = 'make clean case=' + case + ' && make -j 10 OPTIMIZE=yes \
+            DEBUG=no case=' + case
+      rc = subprocess.call(cmd, shell=True)
+      
+       # Check for error code
+      if rc != 0:
+        print('\nERROR: Failed to build the execute file. Check the Makefile \
+              or source code\n')
+        os._exit(0)
+
+      
+      
+      os.chdir(pwd)
+      
+      # 1. Running in uniform CMFD with setLatticeStructure, without MPI
+      cmd = """mpiexec -n 1 ./run_time_standard -domain_decompose 1,1,1 \
```

Comment:
We'll need to add some stuff to the travis.sh script if we want to test the MPI build like that

Reply:
I don't know much about tavis.sh